packages/shared/src/utils/login/login.ts

```diff
@@ -15,11 +15,19 @@ export const redirectToLogin = (is_logged_in, language, has_params = true, redir
     }
 };
 
-export const redirectToSignUp = ({ is_appstore }) => {
+type TRedirectToSignUp = {
+    is_appstore: boolean;
+};
+
+export const redirectToSignUp = ({ is_appstore }: TRedirectToSignUp) => {
```

Comment:
as I see in the code, this function also can take {is_deriv_crypto}
please check